set.mm

```diff
@@ -662642,6 +663452,1224 @@ closed under the multiplication ( ' X ' ) of a finite number of
+      ffnd wbr uzred ltpnf elicod adantl fnfvima2 ffvelrnda elind ne0i ad2antrr
+      syl ex reximdva ralimdva mpd liminflelimsup ) AHIBJADKJBGDJLADCUAFUBMUCAH
+      NZDLZHINZOPQZRZISTBVQOPQZUDZKUEZUFUGZHVTRZISTAIHCDEFUHAWAWFISAVSSLZUIVRWE
+      HVTAVRWEUJWGVQVTLAVRWEAVRUIZVQBUKZWDLWEWHWCKWIWHDVQWBBABDULVRADKBGUTUMAVR
+      UNVRVQWBLAVRVQOVQVRVQCDFVRUOZUPZOKLVRUQMWKVRVQWKURVRVQSLVQOUSVAVRVQCDFWJV
+      BVQVCVKVDVEVFADKVQBGVGVHWDWIVIVKVLVJVMVNVOVP $.
+  $}
+
+  ${
+    $d A x $.
+    liminfvalxrmpt.1 $e |- F/ x ph $.
+    liminfvalxrmpt.2 $e |- ( ph -> A e. V ) $.
+    liminfvalxrmpt.3 $e |- ( ( ph /\ x e. A ) -> B e. RR* ) $.
+    $( Alternate definition of ` liminf ` when ` F ` is an extended real valued
+       Function.  (Contributed by Glauco Siliprandi, 2-Jan-2022.) $)
+    liminfvalxrmpt $p |- ( ph -> ( liminf ` ( x e. A |-> B ) ) = -e ( limsup `
+       ( x e. A |-> -e B ) ) ) $=
+      ( cmpt clsi cfv cv cxne clsp nfmpt1 cxr fmptd2f liminfvalxr wcel xnegeqd
+      wa eqidd fvmpt2d mpteq2da fveq2d eqtrd ) ABCDIZJKBCBLZUGKZMZIZNKZMBCDMZIZ
+      NKZMABCUGEBCDOGABCDPFHQRAULUOAUKUNNABCUJUMFAUHCSUAUIDABCDUGPAUGUBHUCTUDUE
+      TUF $.
+  $}
+
+  ${
+    liminfresuz2.1 $e |- ( ph -> M e. ZZ ) $.
+    liminfresuz2.2 $e |- Z = ( ZZ>= ` M ) $.
+    liminfresuz2.3 $e |- ( ph -> F e. V ) $.
+    liminfresuz2.4 $e |- ( ph -> dom F C_ ZZ ) $.
+    $( If the domain of a function is a subset of the integers, the inferior
+       limit doesn't change when the function is restricted to an upper set of
+       integers.  (Contributed by Glauco Siliprandi, 2-Jan-2022.) $)
+    liminfresuz2 $p |- ( ph -> ( liminf ` ( F |` Z ) ) = ( liminf ` F ) ) $=
+      ( cr cres cdm cz wss dmresss a1i sstrd liminfresuz ) ABCDEFGHABJKLZBLZMST
+      NABJOPIQR $.
+  $}
+
+  ${
+    liminfgelimsupuz.1 $e |- ( ph -> M e. ZZ ) $.
+    liminfgelimsupuz.2 $e |- Z = ( ZZ>= ` M ) $.
+    liminfgelimsupuz.3 $e |- ( ph -> F : Z --> RR ) $.
+    $( The inferior limit is greater than or equal to the superior limit if and
+       only if they are equal.  (Contributed by Glauco Siliprandi,
+       2-Jan-2022.) $)
+    liminfgelimsupuz $p |- ( ph -> ( ( limsup ` F ) <_ ( liminf ` F ) <-> (
+       liminf ` F ) = ( limsup ` F ) ) ) $=
+      ( clsp cfv clsi cle wbr wceq wa cxr wcel cvv cr cuz adantr fvexi a1i fexd
+      liminfcld limsupcld frexr liminflelimsupuz xrletrid eqcomd adantl xreqled
+      simpr id impbida ) ABHIZBJIZKLZUPUOMZAUQNUPUOAUPOPUQABQADRQBGDQPADCSFUAUB
+      UCZUDTAUOOPZUQABQUSUEZTAUPUOKLUQABCDEFADBGUFUGTAUQULUHAURNUOUPAUTURVATURU
+      OUPMAURUPUOURUMUIUJUKUN $.
+  $}
+
+  ${
+    $d A x $.  $d M x $.
+    liminfval4.x $e |- F/ x ph $.
+    liminfval4.a $e |- ( ph -> A e. V ) $.
+    liminfval4.m $e |- ( ph -> M e. RR ) $.
+    liminfval4.b $e |- ( ( ph /\ x e. ( A i^i ( M [,) +oo ) ) ) -> B e. RR )
+                  $.
+    $( Alternate definition of ` liminf ` when the given a function is
```

Comment:
when the given function

Reply:
fixed in e5878437 (thank you for the revision!)